src/pages/store/orders.tsx

```diff
@@ -1,26 +1,79 @@
-import { Navbar } from '@/components/store';
+import { Dropdown, Typography } from '@/components/common';
+import { Navbar, OrdersDisplay } from '@/components/store';
+import { StoreAPI } from '@/lib/api';
 import withAccessType from '@/lib/hoc/withAccessType';
-import { PermissionService } from '@/lib/services';
-import { PrivateProfile } from '@/lib/types/apiResponses';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PrivateProfile, PublicOrder } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import styles from '@/styles/pages/store/orders/index.module.scss';
```

Comment:
```suggestion
import styles from '@/styles/pages/StoreOrders.module.scss';
```

this is maybe just a personal design preference but I don't see any reason for page CSS sheets to match the name of the page file, which has to be named to define the navigation route. I think it's easier for developers a tiny bit to work with